docs/website/docs/ml-frameworks/tensorflow-lite.md

```diff
@@ -1,14 +1,61 @@
 # TensorFlow Lite Integration
 
-!!! todo
-    [Issue#5455](https://github.com/google/iree/issues/5455): write this documentation
+IREE supports compiling and running pre-trained TFLite models.  It converts a model to [TOSA MLIR](https://mlir.llvm.org/docs/Dialects/TOSA/), then compiles it into a VM module.
+
+## Prerequisites
+
+Download a pre-trained TFLite model from the list of [hosted models](https://www.tensorflow.org/lite/guide/hosted_models).
```

Comment:
mention the TensorFlow Lite converter
also a few nits: wrap lines at 80 characters and spell out "TensorFlow Lite"

```suggestion
Download a pre-trained TensorFlow Lite model from the list of
[hosted models](https://www.tensorflow.org/lite/guide/hosted_models)
or convert your own model to a TensorFlow Lite model using the
[TensorFlow Lite converter](https://www.tensorflow.org/lite/convert).
```